message acquire {
    required uint32 version = 1;
    optional string device_bigid = 2;
}

message acquireresponse {
    required uint32 version = 1;
    optional bool success = 2;
    optional string message = 3;
    optional string error_type = 4;
    optional string tuning_bigid = 5;
}
